lib/db/scope.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const unbuf = require('buf').unbuf.hex;
+
+const Scope = {
```

Comment:
@divyabiyani does it make sense to call this `helpers` or something? and rename this to `lib/db/helpers.js`?

We also need a unit test for every function in this module